File: ruby/5_kyu/cw_hashtag.rb
def generateHashtag(str)
  word = str.split(" ").map(&:capitalize)
  comboword = word.join("")
  if comboword.length == 0 || comboword.length > 139
    return false
  else
    return "#" + comboword
  end
end

# https://www.codewars.com/kata/52449b062fb80683ec000024

File: javascript/5_kyu/cw_hashtag.js
function getHashtags(post) {
  let arr = post.split(" ");
  return arr.filter(word => /(?<=^\#+)\w+$/.test(word)).map(word => word.replace(/\#/g, ""));
}

// https://www.codewars.com/kata/52ae2db783f47875d0000064/train/javascript